### PR TITLE
introduce pre-commit typos check

### PR DESCRIPTION
**Pull request by anthonywu**
@filipstrand I see you've already done some of this cleanup in #70, but here's the formal introduction to the repo.

### CODE REVIEW

File: src/mflux/models/transformer/transformer.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 from mflux.config.model_config import ModelConfig
 from mflux.config.runtime_config import RuntimeConfig
-from mflux.models.transformer.ada_layer_norm_continous import (
+from mflux.models.transformer.ada_layer_norm_continuous import (
```
Comment (anthonywu) [inline src/mflux/models/transformer/transformer.py:8]: file is renamed without modifications, happy to find a low friction fix here

File: _typos.toml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# this is the config file for typos-cli, part of "pre-commit run"
+# use "typos --dump-config" to disocver other configs to override
+
+[default]
+extend-ignore-identifiers-re = ["EmbedND", "embed_nd", "alle"]
```
Comment (anthonywu) [inline _typos.toml:5]: `alle` the Italian word from README. IMO not worth trying to introduce non-`en` locale settings. Just exempt this.

Comment (filipstrand) [reply _typos.toml:5]: Agree

**Comment by filipstrand** [comment]
Nice tool to have!

**Pull request merged**